### DISCUSSION TITLE
Is there any way to Ignore http header in micronaut framework

### DISCUSSION THREAD

**Discussion by sduvvada** [micronaut-projects/micronaut-core#6828] [Q&A]
Team,
I'm new to micronaut framework, Is there any way to ignore the http request header like `x-http-method-override` in framework?
If yes, please direct me some sample example or links to refer it.

Thank you!

**Comment by jameskleeh** [comment]
Ignore in what way? Micronaut doesn't look for that header afaik

**Reply by sduvvada** [reply]
Thank you for quick reply James! Let say if client pass this header or in general to ignore any header from the request. Does micronaut have any option to do that using configuration.

**Reply by jameskleeh** [reply]
What does it mean to ignore it? Why does it matter if the header is present in the request or not?

**Reply by jameskleeh** [reply]
I think the answer in general is no
